src/component/registry/types.ts

```diff
@@ -54,4 +54,19 @@ export type OverlayPosition = OverlayShapePosition | OverlayEdgePosition;
 export interface Overlay {
   position: OverlayPosition;
   label?: string;
+  style?: {
+    font?: {
+      color?: string;
+      size?: number;
+    };
+    fill?: {
+      color?: string;
+      opacity?: number;
+    };
+    stroke?: {
+      color?: string;
+      pattern?: string;
```

Comment:
⚠️ we must document how to set the pattern, this is not obvious as not related to a CSS property

Reply:
we have dropped the pattern options, so not relevant anymore.